.github/workflows/release.yml

```diff
@@ -158,6 +160,7 @@ jobs:
           name: arduino-create-agent-${{ matrix.os }}${{ matrix.arch }}
           path: |
             arduino-create-agent*
+            !arduino-create-agent_cli*
```

Comment:
This is a breaking change. We do not ship in future releases the `_cli` binary. It will be used only to run the e2e tests